src/Grid.php

```diff
@@ -138,6 +138,67 @@ public function addButton($text)
         return $this->menu->addItem()->add(new Button($text));
     }
 
+    /**
+     * Add Dropdown menu in grid menu in order to dynamically setup number of item per page.
+     *
+     * @param array  $items An array of item's per page value.
+     * @param null   $ipp   The default item's per page value.
+     * @param string $label The memu item label.
+     *
+     * @throws Exception
+     *
+     * @return $this
+     */
+    public function addRowLimiter($items = [10, 25, 50, 100], $ipp = null, $label = 'Items per page')
```

Comment:
Use it here since this was the only way to setup the number of item correctly inside button menu. Otherwise, if set separately then I need to set menu label at renderHtml() and did not want to add extra method in Grid.